static/js/tools/shared/vis_tools/landing_page_example_links.ts

```diff
@@ -0,0 +1,129 @@
+ *  the map, scatter, and timeline tools.
+ */
+
+interface LandingPageLinkConfig {
+  mapLinks: Link[];
+  scatterLinks: Link[];
+  timelineLinks: Link[];
+}
+
+export const landingPageLinks: LandingPageLinkConfig = {
+  mapLinks: [
+    {
+      id: "map_water_withdrawal_rate_in_the_usa",
+      title: intl.formatMessage({
+        id: "water_withdrawal_rate_in_the_usa",
+        defaultMessage: "Water withdrawal rate in the USA",
+        description:
+          "Title of a map plotting the statistical variable 'water withdrawal rates' in US states.",
+      }),
+      url: "/tools/map#%26sv%3DWithdrawalRate_Water%26pc%3D0%26denom%3DCount_Person%26pd%3Dcountry%2FUSA%26ept%3DCounty",
+    },
+    {
+      id: "map_unemployment_rate_in_new_jersey",
+      title: intl.formatMessage({
+        id: "unemployment_rate_in_new_jersey",
+        defaultMessage: "Unemployment rate in New Jersey",
+        description:
+          "Title of a map plotting the statistical variable 'unemployment rate' in counties of New Jersey",
+      }),
+      url: "/tools/map#%26sv%3DUnemploymentRate_Person%26pc%3D0%26denom%3DCount_Person%26pd%3DgeoId%2F34%26ept%3DCounty",
+    },
+    {
+      id: "map_median_income_in_texas",
+      title: intl.formatMessage({
+        id: "median_income_in_texas",
+        defaultMessage: "Median Income in Texas",
+        description:
+          "Title of a map plotting the statistical variable 'median income' in counties of Texas",
+      }),
+      url: "/tools/map#%26sv%3DMedian_Income_Person%26pc%3D0%26denom%3DCount_Person%26pd%3DgeoId%2F48%26ept%3DCounty",
+    },
+    {
+      id: "map_attainment_of_bachelor_degree_in_colorado",
+      title: intl.formatMessage({
+        id: "attainment_of_bachelor_degree_in_colorado",
+        defaultMessage: "Attainment of bachelor's degree in Colorado",
+        description:
+          "Title of a map plotting the statistical variable 'bachelors degree attainment' for counties of Colorado",
+      }),
+      url: "/tools/map#%26sv%3DCount_Person_EducationalAttainmentBachelorsDegreeOrHigher%26pc%3D1%26denom%3DCount_Person%26pd%3DgeoId%2F08%26ept%3DCounty",
+    },
+  ],
+  scatterLinks: [
+    {
+      id: "scatter_prevalence_of_coronary_heart_disease_vs_projected_temperature_rise_in_the_usa",
+      title: intl.formatMessage({
+        id: "prevalence_of_coronary_heart_disease_vs_projected_temperature_rise_in_the_usa",
+        defaultMessage:
+          "Prevalence of coronary heart disease vs. projected temperature rise in the USA",
+        description:
+          "Title of a scatter plot showing the correlation between coronary heart disease and projected temperature rise in the USA",
+      }),
+      url: "/tools/scatter#%26svx%3DDifferenceRelativeToBaseDate2006_Max_Temperature_RCP45%26dx%3DCount_Person%26svy%3DPercent_Person_WithCoronaryHeartDisease%26dy%3DCount_Person%26epd%3Dcountry%2FUSA%26ept%3DCounty%26qd%3D1%26dd%3D1%26pp%3D",
```

Comment:
Note to reviewers: Codacy is flagging these long urls, but `./run_test -f` requires me to format these as is to pass our lint tests.